src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -31,7 +31,8 @@ set MSBUILD_PROJECT_CONTENT= ^
 
 set PUBLISH_TFM=netcoreapp2.0
 
-set INIT_TOOLS_RESTORE_ARGS=--source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json %INIT_TOOLS_RESTORE_ARGS%
+set DEFAULT_RESTORE_ARGS=--no-cache --packages "%PACKAGES_DIR%"
```

Comment:
Specified `--no-cache` because that seems to be the default for the CoreFX restores done outside this init-tools.cmd